crates/engarde/src/syntax.rs

```diff
@@ -19,6 +20,12 @@ impl Syntax {
         }
     }
 
+    pub fn load_custom_syntaxes(&mut self, syntaxes_path: &Path) {
```

Comment:
Raw needs this function as well so they can be used interchangeably

Reply:
Sure! Should that just be a no-op? Or should it log a warning or something?

Reply:
Let's leave it as a no-op